src/main/java/it/gov/pagopa/payhub/ionotification/constants/IONotificationConstants.java

```diff
@@ -4,6 +4,10 @@ public class IONotificationConstants {
     public static final String SERVICE_STATUS_CREATED = "CREATED";
     public static final String SERVICE_STATUS_REQUESTED = "REQUESTED";
     public static final String SERVICE_STATUS_DELETED = "DELETED";
+    public static final String NOTIFICATION_STATUS_OK = "OK";
+    public static final String NOTIFICATION_STATUS_KO_SERVICE_NOT_FOUND = "KO_SERVICE_NOT_FOUND";
+    public static final String NOTIFICATION_STATUS_KO_SENDER_NOT_ALLOWED = "KO_SENDER_NOT_ALLOWED";
```

Comment:
what about grouping those different constants representing multiple status into Enums to logically group together a set of status values for an entity?